### PR TITLE
Update v1alpha2 sdk and notebook

### PR DESCRIPTION
**Pull request by yuzisun**
**What this PR does / why we need it**:
- update sdk to latest v1alpha2 API
- update notebook example

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. tested the sdk notebook example

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/350)

### CODE REVIEW

File: python/kfserving/kfserving/models/v1alpha2_model_spec.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2019 kubeflow.org.
```
Comment (jinchihe) [inline python/kfserving/kfserving/models/v1alpha2_model_spec.py]: Removed this files?

File: python/kfserving/kfserving/models/v1alpha2_alibi_explain_spec.py [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2019 kubeflow.org.
```
Comment (jinchihe) [inline python/kfserving/kfserving/models/v1alpha2_alibi_explain_spec.py]: Removed this file?

File: python/kfserving/kfserving/models/v1alpha2_explain_spec.py [hunk 3]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2019 kubeflow.org.
```
Comment (jinchihe) [inline python/kfserving/kfserving/models/v1alpha2_explain_spec.py]: Remove this file?

File: docs/samples/client/kfserving_sdk_sample.ipynb [hunk 4]
```diff
@@ -77,9 +78,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (jinchihe) [inline docs/samples/client/kfserving_sdk_sample.ipynb]: Not sure if we should clean up the cell outputs for easy reviewing, this is optional :-)

File: python/kfserving/kfserving/models/v1alpha2_model_spec.py (hunk 1 above)
Comment (yuzisun) [reply python/kfserving/kfserving/models/v1alpha2_model_spec.py]: removed

File: python/kfserving/kfserving/models/v1alpha2_alibi_explain_spec.py (hunk 2 above)
Comment (yuzisun) [reply python/kfserving/kfserving/models/v1alpha2_alibi_explain_spec.py]: removed

File: python/kfserving/kfserving/models/v1alpha2_explain_spec.py (hunk 3 above)
Comment (yuzisun) [reply python/kfserving/kfserving/models/v1alpha2_explain_spec.py]: removed

File: docs/samples/client/kfserving_sdk_sample.ipynb (hunk 4 above)
Comment (yuzisun) [reply docs/samples/client/kfserving_sdk_sample.ipynb]: cleared the output

**Comment by yuzisun** [comment]
> Thanks @yuzisun
> While we are at it, would it make sense to extend the SDK notebook to show preprocessing/postprocessing/explainability in action through the SDK?

yes I am creating separate notebooks for transformer and explainer, this notebook is for canary rollout.

**Comment by jinchihe** [comment]
Thanks @yuzisun 
/lgtm

**Pull request merged**